src/renderers/webgl/WebGLProgram.js

```diff
@@ -152,7 +152,7 @@ function replaceLightNums( string, parameters ) {
 
 function parseIncludes( string ) {
 
-	var pattern = /#include +<([\w\d.]+)>/g;
+	var pattern = /^\s*#include +<([\w\d.]+)>/gm;
```

Comment:
g searches for more then 1 result.
m makes it so that ^ means start of line instead of start of string.